erpnext/accounts/doctype/payment_entry/test_payment_entry.py

```diff
@@ -107,7 +107,7 @@ def test_payment_entry_against_si_usd_to_usd(self):
 		pe = get_payment_entry("Sales Invoice", si.name, bank_account="_Test Bank USD - _TC")
 		pe.reference_no = "1"
 		pe.reference_date = "2016-01-01"
-		pe.target_exchange_rate = 50
+		pe.source_exchange_rate = 50
```

Comment:
Ideally, it should work irrespective of we set source_exchange_rate or target_exchange_rate.

Reply:
There should be a base or reference for this, right? User may change the exchange rate manually and the visible option to the user in the form is source hence I used that as a reference. For test cases, we have to set one so that the values we are checking in test match the expected value.